src/commands/create_account.js

```diff
@@ -14,31 +14,55 @@
  *
  */
 import cryptography from '../utils/cryptography';
+import { ValidationError } from '../utils/error';
 import { createCommand } from '../utils/helpers';
 import { createMnemonicPassphrase } from '../utils/mnemonic';
+import commonOptions from '../utils/options';
 
-const description = `Returns a randomly-generated mnemonic passphrase with its corresponding public key and address.
+const description = `Returns a list of randomly-generated mnemonic passphrase with its corresponding public key and address.
 
-	Example: create account
+	Examples:
+	- create account
+	- create account -n 50
 `;
 
-export const actionCreator = () => async () => {
-	const passphrase = createMnemonicPassphrase();
-	const { privateKey, publicKey } = cryptography.getKeys(passphrase);
-	const { address } = cryptography.getAddressFromPublicKey(publicKey);
-
-	return {
-		passphrase,
-		privateKey,
-		publicKey,
-		address,
-	};
+const checkIsInt = number => Number.isInteger(parseInt(number, 10));
+
+const checkIsNonZeroInt = number => parseInt(number, 10) !== 0;
+
+export const actionCreator = () => async ({ options }) => {
+	const { number = 1 } = options;
+
+	if (!checkIsInt(number)) {
+		throw new ValidationError(
+			`${number} doesn't represent a valid Integer corresponding to number of accounts`,
+		);
+	}
+
+	if (!checkIsNonZeroInt(number)) {
```

Comment:
I think nonzero interger doesn't mean it is positive number

Reply:
I am replacing the nonzero check with checkIsPositiveInt.

Reply:
It looks like: const checkIsPositiveInt = number => parseInt(number, 10) > 0;